### ISSUE TITLE
Update @typechain/ethers-v5 to v6

### ISSUE THREAD

**Issue by jacque006** [thehubbleproject/hubble-contracts#564]
# @typechain/ethers-v5
https://www.npmjs.com/package/@typechain/ethers-v5
Update to latest v6

**Comment by ChihChengLiang** [comment]
- We could delay the upgrade of `mcl-wasm`, as hubble-contracts will not directly depend on mcl-wasm after #565.
- We shouldn't upgrade Solidity to v0.8 right now. The contracts are audited, upgrading to Solidity v0.8 will bring breaking changes and unexpected behaviors.

**Comment by jacque006** [comment]
Thanks for input @ChihChengLiang , removed `mcl-wasm` in favor of https://github.com/thehubbleproject/hubble-contracts/issues/565 and moved Solidity update to a delayed section.

I will likely break more of this up next week.

**Comment by jacque006** [comment]
Split this up into multiple issues so work is better broken up.

- OpenZepplin/Solidity v0.8 https://github.com/thehubbleproject/hubble-contracts/issues/566
- NodeJS v14 https://github.com/thehubbleproject/hubble-contracts/issues/567
- Prettier v2 https://github.com/thehubbleproject/hubble-contracts/issues/568
